packages/build/src/plugins_core/functions/index.js

```diff
@@ -7,15 +7,34 @@ const pathExists = require('path-exists')
 const { isDirectory } = require('path-type')
 
 const { addErrorInfo } = require('../../error/info')
-const { logFunctionsNonExistingDir, logFunctionsToBundle } = require('../../log/messages/core_commands')
+const {
+  logExperimentalEsbuildParameter,
+  logFunctionsNonExistingDir,
+  logFunctionsToBundle,
+} = require('../../log/messages/core_commands')
 
 const { getZipError } = require('./error')
 
+const getZISIParameters = ({ childEnv, logs, netlifyConfig }) => {
```

Comment:
@ehmicky I ended up breaking this part out into a separate function instead. Doing it for lines 39-47 was a bit more complex, due to an early return.

I'm happy to revisit it in a subsequent PR though!

Reply:
This looks good. :+1: